converters/fromZigbee.js

```diff
@@ -5569,6 +5569,49 @@ const converters = {
             return {state: (msg.data['presentValue']==0) ? 'OFF' : 'ON'};
         },
     },
+    neo_t_h_alarm: {
+        cluster: 'manuSpecificTuyaDimmer',
+        type: ['commandSetDataResponse', 'commandGetData'],
+        convert: (model, msg, publish, options, meta) => {
+            const dp = msg.data.dp;
+            const data = msg.data.data;
+            const dataAsDecNumber = utils.convertMultiByteNumberPayloadToSingleDecimalNumber(data);
+            switch (dp) {
+            case 360: // 0x0168 [0]/[1] Alarm!
+                return {alarm: (dataAsDecNumber==0) ? 'OFF' : 'ON'};
```

Comment:
Instead of `OFF`/`ON`, doesn't it make more sense to return `true`/`false`?

Reply:
ok